src/Parser/Parser.hs

```diff
@@ -154,6 +162,10 @@ parens = P.parens lexer
 identifier :: ParsecT String u Identity String
 identifier = P.identifier lexer
 
+-- | Lowercase identifier recognizer
+lowerIdentifier :: ParsecT String u Identity String
```

Comment:
My intial nitpick was that your change reused this expression instead of making it a function. It led me to realize that we accepted reserved words as identifiers in some locations.

Don't tell anyone we made such a silly mistake ;)

Reply:
It's public record now 👍 , but that's part of the dev cycle. Yeah, having a lowerIdentifier akin with our capIdentifier is sensible. And ahhhh.

Reply:
Haha, that comment was meant in jest. In fact, by mentioning it, I probably drew extra attention to it. It's like an intentional [streisand effect](https://en.wikipedia.org/wiki/Streisand_effect).